MrE/src/main/java/frc/robot/commands/DriveScaling.java

```diff
@@ -0,0 +1,35 @@
+package frc.robot.commands;
+
+/**
+ * This enumeration represents the different options available for drive scaling.
+ * 
+ * @author Adam T Poole
+ * @author Bridget Brenner
+ *
+ */
+public enum DriveScaling {
+	
+	CUBIC("Cubic"),
+	QUADRATIC("Linear"),
+	SQUARED("Squared"),
+	LIMITED_POLYNOMIC("Limited Polynomic");
```

Comment:
This enumeration uses an additional parameter that allows you to pass in a human readable name that can be displayed.